aten/src/ATen/native/AveragePool3d.cpp

```diff
@@ -177,21 +177,18 @@ static void avg_pool3d_out_frame(
 {
   at::parallel_for(0, nslices, 0, [&](int64_t start, int64_t end) {
     for (const auto k : c10::irange(start, end)) {
-      // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-      int64_t i, j, ti;
-
       /* local pointers. */
       const scalar_t *ip = input_p + k * itime * iwidth * iheight;
       scalar_t *op = output_p + k * otime * owidth * oheight;
-      for (i = 0; i < otime * oheight * owidth; ++i)
+      for (int i = 0; i < otime * oheight * owidth; ++i)
```

Comment:
Ugh... these were originally int64_t, why the type change?

Reply:
A copy error, let me fix it.